keps/sig-scheduling/5027-dra-admin-controlled-device-attributes/README.md

```diff
@@ -283,6 +271,11 @@ type DevicePatch struct {
     // The name of each attribute must be unique in that set and
     // include the domain prefix.
     //
+    // In contrast to attributes in a ResourceSlice, entries here are allowed to
+    // be marked as empty by setting their null field. Such entries remove the
+    // corresponding attribute in a ResourceSlice, if there is one, instead of
+    // overriding it.
```

Comment:
Is it worth enforcing that null cannot be set in a ResourceSlice? If it's allowed, that would leave the option open for drivers to do so in case they want to communicate some nuance like "I think users may expect this attribute to exist, but it intentionally does not."

Reply:
I think that's too nuanced and adds one more thing that users writing CEL expressions would have to be prepared for, in addition to the "can I access this attribute without getting a lookup error".